### ISSUE TITLE
H36M evaluation

### ISSUE THREAD

**Issue by mimiliaogo** [hongsukchoi/3DCrowdNet_RELEASE#29]
Hi, 
Have you ever evaluated your method on Human3.6M testing set?
Because I am curious about whether your method still works well in simple environments and is comparable to other methods.

**Comment by hongsukchoi** [comment]
@mimiliaogo 

Yes, I remember it worked well. I guess there's no reason for 3DCrowdNet to work worse in simple environments.

**Comment by mimiliaogo** [comment]
@hongsukchoi  
Thank you for getting back to me. But I used your model weight given in [here](https://drive.google.com/drive/folders/19ntGuC0zaXQa3cCN_2Ox_hWYX3nLLP2J ) with version 04-06 snapshot 10, and evaluated it on Human3.6M but got a poor result compared to previous methods:
MPJPE from mesh: 69.22 mm
PA MPJPE from mesh: 44.47 mm

Do you have any idea about this result?

Thank you so much.

**Comment by hongsukchoi** [comment]
@mimiliaogo 

The most likely reason is using wrong 2D pose input. Which 2D pose input did you use?

**Comment by MilkLoo** [comment]
@mimiliaogo 
Hello, I would like to ask you how to write the code for the test and evaluation part of Human3.6M? Can you share? Thank you very much!
2395611610@qq.com
